eclipse/src/saros/ui/eventhandler/ConnectingFailureHandler.java

```diff
@@ -19,66 +18,56 @@
  */
 public class ConnectingFailureHandler implements IConnectingFailureCallback {
 
-  private static final Logger LOG = Logger.getLogger(ConnectingFailureHandler.class);
+  private static final Logger log = Logger.getLogger(ConnectingFailureHandler.class);
 
   private final ConnectionHandler connectionHandler;
-  private final XMPPAccountStore accountStore;
 
   private boolean isHandling;
 
-  public ConnectingFailureHandler(
-      final ConnectionHandler connectionHandler, final XMPPAccountStore accountStore) {
+  public ConnectingFailureHandler(final ConnectionHandler connectionHandler) {
     this.connectionHandler = connectionHandler;
     this.connectionHandler.setCallback(this);
-    this.accountStore = accountStore;
   }
 
   @Override
   public void connectingFailed(final Exception exception) {
-    SWTUtils.runSafeSWTAsync(LOG, () -> handleConnectionFailed(exception));
+    SWTUtils.runSafeSWTAsync(log, () -> handleConnectionFailed(exception));
   }
 
   private void handleConnectionFailed(Exception exception) {
 
-    // account store is empty
-    if (exception == null) {
-      // Wizard will perform a connection attempt if it is finished
-      SWTUtils.runSafeSWTAsync(LOG, () -> WizardUtils.openSarosConfigurationWizard());
-      return;
-    }
+    try {
 
-    // avoid mass dialog popups
-    if (isHandling) return;
+      // avoid mass dialog popups
+      if (isHandling) return;
 
-    try {
       isHandling = true;
 
       if (!(exception instanceof XMPPException)) {
-
-        DialogUtils.popUpFailureMessage(
+        MessageDialog.openError(
+            SWTUtils.getShell(),
             "Connecting Error",
             MessageFormat.format(
-                "Could not connect to XMPP server. Unexpected error: {0}", exception.getMessage()),
-            false);
-
+                "Could not connect to XMPP server. Unexpected error: {0}", exception.getMessage()));
```

Comment:
This should be moved to a messages.properties file as it is text shown to the user.

Reply:
Can we agree that moving strings into property files can be done just before the next release or at least later ? There is automated tooling provided by Eclipse for this which also inserts a bunch of NLS.

Reply:
Ok. How about opening an issue for this? It would also be helpful to link to some documentation on how this can be done automatically as I had no idea this was an option in Eclipse. ☺️

Reply:
The problem is the Google Formatter.

E.g you have "foo0" + "foo1" + .... + "fooN"

When say you do not want to externalize them Eclipse will add something like this // $NON-NLS-1$, $NON-NLS-2$, ... behind this line.

I have not really tested it what will happen if the comment will be split to several lines afterwards.

Reply:
Wow nice job. https://github.com/google/google-java-format/issues/361

Currently there is now way to tell if the string already was externalized or not.